src/windows/Accessibility.ts

```diff
@@ -3,17 +3,41 @@
 *
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT license.
-*
-* Web wrapper for subscribing or querying the current state of the
-* screen reader.
 */
 
-import { Accessibility as CommonAccessibility } from '../common/Accessibility';
+import RN = require('react-native');
+import SubscribableEvent from 'subscribableevent';
+import { Accessibility as NativeAccessibility } from '../native-common/Accessibility';
+import SyncTasks = require('synctasks');
+
+export class Accessibility extends NativeAccessibility {
+    private _isHighContrast: boolean|undefined;
+
+    constructor() {
+        super();
+
+        if (RN.AccessibilityInfo) {
+            RN.AccessibilityInfo.addEventListener('highContrastDidChange', isEnabled => {
+                this._updateIsHighContrast(isEnabled);
+            });
+
+            if (RN.AccessibilityInfo.fetchIsHighContrast) {
+                RN.AccessibilityInfo.fetchIsHighContrast().then(isEnabled => {
+                    this._updateIsHighContrast(isEnabled);
```

Comment:
>this._updateIsHighContrast(isEnabled); [](start = 20, length = 38)

Before calling this, we need to check to make sure we haven't already initialized `_isHighContrast` due to `highContrastDidChange`. Currently, we're at risk of overwriting `_isHighContrast` with a stale value.

See how `_isScreenReaderEnabeld` is initialized here: [https://github.com/antonkh/reactxp/blob/c633597d5f28fd4acecaf1667b1855c4132ddb48/src/native-common/Accessibility.ts](https://github.com/antonkh/reactxp/blob/c633597d5f28fd4acecaf1667b1855c4132ddb48/src/native-common/Accessibility.ts)

It may seem like an unlikely race condition but I've seen it hit and it's a pain to debug.

Reply:
Good point. Fixed.